rtcdatachannel_test.go

```diff
@@ -1,35 +1,31 @@
 package webrtc
 
-import (
```

Comment:
We can keep this test if we move ID stuff back

Reply:
Yeah no problem. I will bring this back for now. But I am ultimately moving that function out of `RTCPeerConnection` since it does not belong there.